src/backend/mailbox_manager/management/commands/setup_dimail_db.py

```diff
@@ -81,6 +81,13 @@ def handle(self, *args, **options):
         else:
             self.create_user(name=people_base_user.sub)
             self.create_allows(people_base_user.sub, domain_name)
+            # create access role for john doe
+            maildomain = MailDomain.objects.get(name=domain_name)
+            MailDomainAccess.objects.create(
+                user=User.objects.get(email="people@people.world"),
+                role=enums.MailDomainRoleChoices.OWNER,
+                domain=maildomain,
+            )
```

Comment:
It should be under the `# we create a domain and add John Doe to it` comment? Because this part is more "people -> dimail"

Also, you should use a `get_or_create` to prevent fail when running several times.